atf_plotter/scripts/plot.py

```diff
@@ -0,0 +1,761 @@
+#!/usr/bin/env python2
```

Comment:
@fmessmer Should we use 
```
#!/usr/bin/env python
```
or explictitly
```
#!/usr/bin/env python3
```
due to the change to python3

Reply:
hmm, we haven't really focused on that aspect during the `pylint` session
personally, I'd vote for version-independent `#!/usr/bin/env python`...but we also have quite a number of python files using `#!/usr/bin/env python2`...

@LoyVanBeek
what's you opinion on this?

once decided, maybe we should be consequent about this and change it everywhere....
I added a reference to this comment to https://github.com/mojin-robotics/cob4/issues/1406

Reply:
It should be just `python` AFAIK and let the distro decide eventually

Reply:
still prefer the aspect "explicit is better than implicit" from python design goals and would let it be python2 until script supports python3.